src/unibasicTemplates/subroutines/internal/setup.njk

```diff
@@ -1,14 +1,18 @@
 subroutine mvom_setup(setupOptions, parsedSetupOptions)
+{% include "../../common/mvom.njk" %}
 {% include "../../constants/setup.njk" %}
 
 * Perform any needed environment setup
 
-com /S$MVOM/ S$MVOM.PROCESS
-com /S$MVOM/ S$MVOM.USER1, S$MVOM.USER2, S$MVOM.USER3, S$MVOM.USER4, S$MVOM.USER5
 S$MVOM.PROCESS = @TRUE
 
 gosub processUserDefinedOptions
 
+S$MVOM.REQUEST.ID = ''
+if udoGetProperty(setupOptions, 'requestId', requestId, type) eq 0 then
```

Comment:
Does `udoGetProperty` return a boolean? Is the logic similar to other languages that 0 is false and otherwise is true?

Just want to understand why the `eq 0` is needed here where I don't see it anywhere else in the repo. Is it for type conversion since requestId is a string?

Reply:
Most (maybe all) of the udo functions return a zero when there is no error, and an error code enumeration otherwise.  The logic elsewhere in this repo is checking to see if there is an error and reacting to that error.  However, this is treating the failure to acquire a requestID as a non-error scenario, so effectively it can be optional.

It's a little more graceful than erroring in this scenario since it doesn't _really_ matter if we don't acquire the request id.

Reply:
Also a quirk with UniBasic, `udoGetProperty(setupOptions, 'requestId', requestId, type)` by itself will not compile. You either need to put it in a conditional or assign the return value to a variable.

Reply:
@jem41 Are you good with this PR?